pkg/operatingsystem/cos/git-reader_test.go

```diff
@@ -0,0 +1,30 @@
+package cos_test
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/storage/memory"
+	"github.com/stretchr/testify/assert"
+	"github.com/thought-machine/falco-probes/pkg/operatingsystem/cos"
+)
+
+func TestReadMilestonesToBuildIDs(t *testing.T) {
```

Comment:
Same comment about the test above.

Reply:
Blah go-git would be super-annoying to mock, they [don't even do it in their own tests](https://github.com/go-git/go-git/blob/master/repository_test.go#L192-L195) so I've put in a TODO for now as suggested.